### PR TITLE
Updated list transfer method url parameter(limit,orderby)

### PR DESCRIPTION
**Pull request by vaprakasam-altmtrk**
Updated list transfer method url parameter(limit,orderby)

### CODE REVIEW

File: transfermethodrepository/src/main/java/com/hyperwallet/android/ui/transfermethod/repository/TransferMethodRepositoryImpl.java [hunk 1]
```diff
@@ -79,6 +79,8 @@ public void createTransferMethod(@NonNull final HyperwalletTransferMethod transf
     public void loadTransferMethods(@NonNull final LoadTransferMethodListCallback callback) {
 
         HyperwalletTransferMethodQueryParam queryParam = new HyperwalletTransferMethodQueryParam.Builder()
+                .limit(100)
```
Comment (peter-joseph) [inline transfermethodrepository/src/main/java/com/hyperwallet/android/ui/transfermethod/repository/TransferMethodRepositoryImpl.java]: can we change this to non literal value? and create constant for example `DEFAULT_LIMIT` or `QUERY_PARAM_DEFAULT_LIMIT` your choice thanks

Comment (peter-joseph) [reply transfermethodrepository/src/main/java/com/hyperwallet/android/ui/transfermethod/repository/TransferMethodRepositoryImpl.java]: and I think its also possible to add unit test, by running a mockwebserver and get request if limit = 100, let me know if you want to know how to do it

Comment (vaprakasam-altmtrk) [reply transfermethodrepository/src/main/java/com/hyperwallet/android/ui/transfermethod/repository/TransferMethodRepositoryImpl.java]: Constant variable created and test case completed for bank account list increased

File: transfermethodrepository/src/main/java/com/hyperwallet/android/ui/transfermethod/repository/TransferMethodRepositoryImpl.java [hunk 2]
```diff
@@ -16,6 +16,7 @@
  */
 package com.hyperwallet.android.ui.transfermethod.repository;
 
+
```
[the remark this answers is not in the document]
Comment (vaprakasam-altmtrk) [reply transfermethodrepository/src/main/java/com/hyperwallet/android/ui/transfermethod/repository/TransferMethodRepositoryImpl.java]: space removed

**Pull request merged**